.github/workflows/release-bin.yml

```diff
@@ -9,9 +9,21 @@ on:
 jobs:
   build:
     name: Rust project
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
     steps:
       - uses: actions/checkout@master
+      - name: Get Toolchain version
```

Comment:
Add a comment here please that explains that this is a workaround until https://github.com/actions/virtual-environments/issues/6 is fixed.